lib/rules/valid-jsdoc.js

```diff
@@ -22,7 +22,8 @@ module.exports = function(context) {
 
         // these both default to true, so you have to explicitly make them false
         requireReturn = options.requireReturn === false ? false : true,
-        requireParamDescription = options.requireParamDescription === false ? false : true;
+        requireParamDescription = options.requireParamDescription === false ? false : true,
```

Comment:
This is a bit hard to read. Just `options.requireParamDescription !==false` is much easier.